### PR TITLE
Bug unclickable space in options

### PR DESCRIPTION
**Pull request by MySmilingTurban**
#228

### CODE REVIEW

**Comment by amitojsingh** [comment]
@MySmilingTurban Veer ji It's not good practice to add a button inside a button.

File: src/components/buttons/Option.tsx [hunk 1]
```diff
@@ -38,28 +38,28 @@ export default function OptionBtn({
...
-      <div className={optionClassname}>
-        <button
-          className={'h-full w-full'}
-          onClick={() => {
-            selector(option);
-            setOptionSelected(true);
-          }}
-          disabled={disabled}
-        >
+      <button
```
Comment (amitojsingh) [inline src/components/buttons/Option.tsx:41]: @MySmilingTurban Veer ji It's not good practice to add a button inside a button.

**Review by amitojsingh — CHANGES_REQUESTED** [review]

File: src/components/buttons/Option.tsx (hunk 1 above)
Comment (MySmilingTurban) [reply src/components/buttons/Option.tsx:41]: do you mean div inside a button veerji?

Comment (amitojsingh) [reply src/components/buttons/Option.tsx:41]: nopes veer ji text to speach button inside button.

Comment (MySmilingTurban) [reply src/components/buttons/Option.tsx]: fixed veerji

**Review by amitojsingh — APPROVED** [review]
LGTM

**Pull request merged**